api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -99,4 +99,8 @@ message QuicProtocolOptions {
   // QUIC layer by replying with an empty version negotiation packet to the
   // client.
   bool reject_new_connections = 13;
+
+  // If true, QUIC connections created by this listener will change IPv6 flow label
```

Comment:
Can you elaborate on the docs for this, and maybe offer hints for when this should or should not be enabled?